index.html

```diff
@@ -787,8 +787,16 @@ <h2>Links</h2>
 
       <p>
         <span class="rfc2119-assertion" id="common-constraints-links-1">The following keywords are defined for the
-          HTTP profiles and MUST be supported by all consumers.</span>
+          HTTP profiles and MAY be present in compliant TDs.</span>
+	<span class="rfc2119-assertion" id="common-constraints-links-1a">Compliant consumers that have the capabilities 
```

Comment:
Again, I'm not sure what a "complaint consumer" means. Maybe you mean. "Consumers that only follows the HTTP Profile, have the capabilities...."